packages/devextreme/js/ui/context_menu/ui.context_menu.js

```diff
@@ -556,6 +555,16 @@ class ContextMenu extends MenuBase {
         if(!arg.cancel) {
             this._hideAllShownSubmenus();
             this._setOptionWithoutOptionChange('visible', false);
+
+            const $scrollableElement = this._overlay.$content().find(`.${SCROLLABLE_CLASS}`).first();
+
+            if(!$scrollableElement.length) {
+                return;
+            }
+
+            const scrollableInstance = $scrollableElement.dxScrollable('instance');
+
+            scrollableInstance.scrollTo(0);
```

Comment:
[missed scenarios]
So, why do you do it only for the 1st level submenu? What about nested submenus? I see it's reproduced when you close submenu by click page other element.